test/runtests.jl

```diff
@@ -6,6 +6,9 @@ using LinearAlgebra
 using Statistics
 using Test, Random, Plots
 using ReferenceTests, ImageIO
+# WARNING: replacing the following line with `using ScientificTypes` will
+# cause a conflicting import and break the tests
+import ScientificTypes: Count, elscitype
```

Comment:
This is strange. Can you please elaborate on how you are running the tests? This shouldn't happen.

Reply:
From the terminal:
```
(base) jafar_isbarov@jafar-IdeaPad-3-15ADA05:~/.julia/dev/TableTransforms/test$ julia runtests.jl
```
  
The reason this is happening is because both `ScientificTypes` and `TypedTables` have a `Table` function.

@juliohm

Reply:
Got it. In this case let's just remove the comment because it is not very helpful.